asmcli/commands/install.sh

```diff
@@ -35,14 +35,19 @@ install() {
 }
 
 install_managed_components() {
+  info "Configuring ASM managed control plane revision CRD..."
+  context_append-kube-yaml "${CRD_CONTROL_PLANE_REVISION}"
+
   info "Configuring base installation for managed control plane..."
   context_append-kube-yaml "${BASE_REL_PATH}"
 
   info "Configuring ASM managed control plane validating webhook config..."
   context_append-kube-yaml "${MANAGED_WEBHOOKS}"
 
-  info "Configuring ASM managed control plane components..."
```

Comment:
so are we going to do anything in the gateway yaml file? Are we clear about the meshconfig settings in that file?

Reply:
For 1.10 I think we're safe, so the v2 branch we can remove it. (If not it'll come up in the bug bash.)